content/blog/2016/2016-04-07-pipeline-for-runs-on-hardware.adoc

```diff
@@ -0,0 +1,258 @@
+---
+layout: post
+title: Automating test runs on hardware with Pipeline-as-Code. Part 1
```

Comment:
"Pipeline as code", no hyphens necessary.

Also, I suggest removing "Part 1" until you actually have "part 2" written.